packages/pds/tests/auth.test.ts

```diff
@@ -1,6 +1,6 @@
 import * as jose from 'jose'
 import { AtpAgent } from '@atproto/api'
-import { TestNetworkNoAppView, SeedClient } from '@atproto/dev-env'
+import { SeedClient, TestNetworkNoAppView } from '@atproto/dev-env'
 import { createRefreshToken } from '../src/account-manager/helpers/auth'
 
 describe('auth', () => {
```

Comment:
It'd be nice to have a quick test in here that verifies that a "takendown" access token isn't able to make normal calls - for instance `createRecord`

Reply:
sure, added the test in `takedown-appeal` which calls createRecord and asserts that `bad token scope` error is thrown.